packages/cli/internal/pkg/cli/workflow/workflow_run_test.go

```diff
@@ -198,6 +203,20 @@ func (s *WorkflowRunTestSuite) TestRunWorkflow_LocalFile_WithS3Args() {
 	s.mockZip.EXPECT().CompressToTmp(testTempDir).Return(testCompressedTmpPath, nil)
 	uploadCall := s.mockS3Client.EXPECT().UploadFile(testOutputBucket, testWorkflowZipKey, testCompressedTmpPath).Return(nil)
 	s.mockStorageClient.EXPECT().ReadAsBytes(testArgumentsPath).Return([]byte(testInputS3), nil)
+	s.mockStorageClient.EXPECT().ReadAsBytes(testMANIFESTPath).Return([]byte(testMANIFEST), nil)
+	eq := gomock.GotFormatterAdapter(
+		gomock.GotFormatterFunc(
+			func(i interface{}) string {
+				return fmt.Sprintf("%s", i)
+			}),
+		gomock.WantFormatter(
+			gomock.StringerFunc(func() string {
+				return fmt.Sprintf("%s", s.testAppendedMANIFEST)
+			}),
+			gomock.Eq([]byte(s.testAppendedMANIFEST)),
+		),
+	)
```

Comment:
qq: could you please explain a bit regarding what this is doing here?

Reply:
Since the Got and Want given by the test is in bytes, this formatter prints it as strings to improve readability in the test. I will refactor to improve readability

Reply:
I see and that make sense.